src/main/java/com/microsoft/aad/msal4j/HttpHeaders.java

```diff
@@ -34,6 +34,9 @@
     private  final static String REQUEST_CORRELATION_ID_IN_RESPONSE_HEADER_NAME = "return-client-request-id";
     private final static String REQUEST_CORRELATION_ID_IN_RESPONSE_HEADER_VALUE = "true";
 
+    private  final static String X_MS_LIB_CAPABILITY_NAME = "x-ms-lib-capability";
```

Comment:
Consider adding a comment on what these 2 capabilities are.

Reply:
going to add